__tests__/integration/handlers/application.js

```diff
@@ -0,0 +1,189 @@
+const applicationHandler = require('../../../src/handlers/applications')
+const { Applications } = require('../../../src/database')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
const { Applications } = require("../../../src/database")
```